ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/HdWallet.hs

```diff
@@ -440,8 +535,8 @@ zoomOrCreateHdRoot :: HdRoot
                    -> HdRootId
                    -> Update' HdRoot    e a
                    -> Update' HdWallets e a
-zoomOrCreateHdRoot newRoot rootId  =
-    zoomCreate newRoot (hdWalletsRoots . at rootId)
+zoomOrCreateHdRoot newRoot rootId upd =
+    zoomCreate newRoot (hdWalletsRoots . at rootId) $ upd
```

Comment:
:man_shrugging:

Reply:
:man_shrugging:

Reply:
The point here is that there are `zoomOrCreateHd{Account,Address}` where you cannot reduce `upd` similarly, so this change makes their implementations more uniform. But I see no point in arguing over this: it is super minor, and if you insist, I'll revert this.

Reply:
> it is super minor

Yes, it is, that's why I wrote :man_shrugging: instead of "wtf fix it pls".

> if you insist

I don't.